R/servers.R

```diff
@@ -154,6 +159,8 @@ findServer <- function(server = NULL,
 #'   character vector containing the certificate's contents.
 #' @param validate Validate that `url` actually points to a Posit Connect
 #'   server?
+#' @param snowflakeConnectionName Name for the Snowflake connection parameters
```

Comment:
Do we want to have a company's name baked into a function argument like this? (Would we expect to do this for AWS, Azure, etc.?) I don't have a ready alternative to suggest, but I wonder if there's a different way to propagate the information.

Reply:
The snowflake connection name indicates the named connection in an individual user's configuration file, which is not the same as the URL.

```toml
[connections.production]
account = "deadbeef-bob-the-builder-com"
```

Here, the connection name is "production". The account value has some bearing on a component of the URL, but I believe that multiple connections could reference the same account (other portions of the configuration would vary). The target URL might be `https://decafbad-deadbeef-bob-the-builder-com.snowflakecomputing.app/` for customer "bob the builder dot com".